terminus-ui/expansion-panel/src/accordion/accordion-base.ts

```diff
@@ -0,0 +1,31 @@
+import { InjectionToken } from '@angular/core';
+import { CdkAccordion } from '@angular/cdk/accordion';
+
+
+/**
+ * Base interface for a {@link TsAccordionComponent}
+ */
+export interface TsAccordionBase extends CdkAccordion {
```

Comment:
why didn't you use `MatAccordionBase`?

Reply:
We aren't using any material for this component (besides some lifted code). Angular Material builds their accordion using the CDK and I just did the same thing.

This follows the same basic pattern we took with the select; extending the CDK directly. The long-term goal is to remove our dependency on Angular Material which will give us greater flexibility as we continue to define our own brand and needs through the library.